src/workerd/jsg/modules.c++

```diff
@@ -371,7 +371,7 @@ v8::Local<v8::Module> compileEsmModule(
     // TODO(later): Use of newExternalOneByteString here limits our built-in source
     // modules (for which this path is used) to only the latin1 character set. We
     // may need to revisit that to import built-ins as UTF-16 (two-byte).
-    contentStr = jsg::check(jsg::newExternalOneByteString(js, content));
+    contentStr = jsg::newExternalOneByteString(js, content);
```

Comment:
why is the `jsg::check` no longer necessary?

Reply:
The PR moves the jsg::check into the implementaiton of `newExternalOneByteString` to simplify